docs/ChainOrderTransitivity.md

```diff
@@ -0,0 +1,179 @@
+# Transitivity of the chain order
+
+Remember that a relation `~` is _transitive_ if `a ~ b` and `b ~ c` implies `a ~ c` for all `a,b,c`.
+It is _non-transitive_ if this is not true, ie if there exist `a,b,c` such that `a ~ b` and `b ~ c`, but not `a ~ c`.
+
+## Context
+
+The abstract Consensus layer currently implements the chain order like this:
+From every header, one can extract a `SelectView`, which has an `Ord` instance.
+A chain `C` is _preferable_ to a chain `C'` if `C'` is empty or if `selectView(H) > selectView(H')` where `H` and `H'` are the tip headers of `C` and `C'`, respectively.
+
+## Examples for non-transitive orders
+
+### Current chain order is non-transitive
+
+For all Shelley-based eras, we currently use a lexicographic-ish[^lexicographic-ish] combination of the following comparisons in descending order:
+
+ 1. Chain length, preferring longer chains.
+ 2. If the issuer identity is the same, compare by the issue/opcert number, preferring higher values, otherwise, no preference.
+ 3. VRF tiebreaker[^vrf-tpraos-vs-praos], preferring lower values.
+
+This is not a total order: The relation is reflexive, antisymmetric and total, but non-transitive.
+
+Consider the following three `SelectView`s:
+
+|              | A | B | C |
+| ------------ | - | - | - |
+| Chain length | l | l | l |
+| Issuer       | x | y | x |
+| Opcert no    | 2 | o | 1 |
+| VRF          | 3 | 2 | 1 |
+
+Lower-case letters stand for arbitrary values (but if they occur repeatedly, they designate the same value).
```

Comment:
"... and distinct variables designate non-equal values"

Reply:
Thanks, fixed:

> Lower-case letters stand for arbitrary values (two letters designate the same value if and only if they are same letter).